### ISSUE TITLE
Declare supported Python versions in setup.{py,cfg}

### ISSUE THREAD

**Issue by Freso** [bmc/munkres#31]
Right now Python support is only checked for using custom code in `setup.py` (the `if sys.version_info[0:2] < (3, 5):` clause). This causes some [issues with downstream users](https://github.com/beetbox/beets/pull/3146) as Python 2.7 and 3.5 were dropped with 1.1.x and just asking for "munkres" will install the latest (1.1.2) instead of the latest for the supported Python version (1.1.2 *or* 1.0.12).

https://packaging.python.org/guides/dropping-older-python-versions/ documents best practices for dropping support for given Python versions. It's too late for the drop of Python 2.7 and 3.5 but should be implemented before Python 3.6 gets dropped.

**Comment by epeisach** [comment]
I agree.  I just ran into this issue. Yes having setup.py tell you there is an issue is not sufficient. On python 2, "pip list -o" implies can update to 1.1.2 - which it allowed.

Under classifiers in setup.py you can add:

'Programming Language :: Python :: 3',

or do the python_requires as mentioned in the URL.
